ordered_set.py

```diff
@@ -335,15 +337,15 @@ def union(self, *sets: SetLike[T]) -> "OrderedSet[T]":
         Each items order is defined by its first appearance.
 
         Example:
-            >>> oset = OrderedSet.union(OrderedSet([3, 1, 4, 1, 5]), [1, 3], [2, 0])
+            >>> oset = OrderedSet([3, 1, 4, 1, 5]).union([1, 3], [2, 0])
```

Comment:
This looked incorrect before because `.union` is a method and not a static or class method. Lmk if I should revert this.